core/templates/pages/about-page/about-page.constants.ts

```diff
@@ -16,13 +16,13 @@
  * @fileoverview Constants for the Oppia about page.
  */
 
-export class AboutPageConstants {
+export const AboutPageConstants = {
   // These credits should not be changed directly. If any change is
   // made to structure/formatting, the respective constants
   // CREDITS_START_LINE, CREDITS_END_LINE and CREDITS_INDENT should be
   // updated in update_changelog_and_credits.py file. If the patterns do
   // not match, update_changelog_and_credits_test will fail.
-  public static CREDITS_CONSTANTS = [
+  CREDITS_CONSTANTS: [
```

Comment:
You will need to do some changes in https://github.com/oppia/oppia/blob/b8d00855e19f15a452a7d60357f0205085562338/scripts/release_scripts/update_changelog_and_credits.py#L55

Reply:
done